schema.json

```diff
@@ -147,9 +144,5 @@
         "oneOf": { "$ref": "#/definitions/schemaArray" },
         "not": { "$ref": "#" }
     },
-    "dependencies": {
-        "exclusiveMaximum": [ "maximum" ],
-        "exclusiveMinimum": [ "minimum" ]
-    },
     "default": {}
```

Comment:
I gotta say, these solutions make me feel a lot better about pushing this forward, and how well JSON Schema is designed in general.